compiler-rt/lib/scudo/standalone/combined.h

```diff
@@ -1259,27 +1275,22 @@ class Allocator {
     storeEndMarker(RoundNewPtr, NewSize, BlockEnd);
   }
 
-  StackDepot *getDepotIfEnabled(const Options &Options) {
-    if (!UNLIKELY(Options.get(OptionBit::TrackAllocationStacks)))
-      return nullptr;
-    return Depot;
-  }
-
   void storePrimaryAllocationStackMaybe(const Options &Options, void *Ptr) {
-    auto *Depot = getDepotIfEnabled(Options);
-    if (!Depot)
+    auto *RingBuffer = getRingBufferIfEnabled(Options);
+    if (!RingBuffer)
       return;
     auto *Ptr32 = reinterpret_cast<u32 *>(Ptr);
-    Ptr32[MemTagAllocationTraceIndex] = collectStackTrace(Depot);
+    Ptr32[MemTagAllocationTraceIndex] = collectStackTrace(RingBuffer->Depot);
     Ptr32[MemTagAllocationTidIndex] = getThreadID();
   }
 
-  void storeRingBufferEntry(void *Ptr, u32 AllocationTrace, u32 AllocationTid,
+  void storeRingBufferEntry(AllocationRingBuffer *RB, void *Ptr,
```

Comment:
We have the var names `RB` and `RingBuffer` in different places for `AllocationRingBuffer`, besides, we still have `RawRingBuffer` when access the entry. Can we rename this to make it less confusing? For example, we could name 

```
RingBuffer => RawRingBuffer
Var of AllocationRingBuffer => RingBuffer
RawRingBuffer => RingBufferEntryAddress (or EntryAddress)
```

Reply:
I suggest we change to

```
RingBuffer => RingBufferPtr
Var of AllocationRingBuffer => RB
RawRingBuffer => RBEntryAddress (and do the indexing in the same line)
```

I think "raw" is a bit confusing, especially as it meant something else before.

Reply:
I see. The new naming looks find to me. Once we can make them more consistent and less confusing.